### PR TITLE
Supports setting the vfox configuration path via the `VFOX_HOME` environment variable

### PR DESCRIPTION
**Pull request by Chance-fyi**
close: #45

### CODE REVIEW

File: internal/path.go [hunk 1]
```diff
@@ -44,14 +44,17 @@ type PathMeta struct {
 }
 
 func newPathMeta() (*PathMeta, error) {
-	userHomeDir, err := os.UserHomeDir()
-	if err != nil {
-		return nil, fmt.Errorf("get user home dir error: %w", err)
+	configPath := os.Getenv("VFOX_CONFIG")
```
Comment (bytemain) [inline internal/path.go:47]: how about `VFOX_CONFIG_HOME`?

Comment (bytemain) [reply internal/path.go:47]: or just `VFOX_HOME`

Comment (aooohan) [reply internal/path.go:47]: I personally prefer `VFOX_HOME`. ;)

**Comment by aooohan** [comment]
This also involves the issue of whether there are permissions. In addition, The default permissions granted now are 0755. Since we are allowed to change `VFOX_HOME`, we should also consider the permissions.

**Comment by Chance-fyi** [comment]
> This also involves the issue of whether there are permissions. In addition, The default permissions granted now are 0755. Since we are allowed to change `VFOX_HOME`, we should also consider the permissions.

This is a problem, I will add a check for `VFOX_HOME` permissions.

**Comment by Chance-fyi** [comment]
Regarding permissions, currently, only errors related to insufficient permissions are being returned.

As for #51, regarding the potential support for multiple users sharing the `VFOX_HOME` directory, this would require setting permissions to at least 775. We need to discuss this matter further, perhaps it could be addressed in another PR.

**Comment by aooohan** [comment]
> Regarding permissions, currently, only errors related to insufficient permissions are being returned.
> 
> As for #51, regarding the potential support for multiple users sharing the `VFOX_HOME` directory, this would require setting permissions to at least 775. We need to discuss this matter further, perhaps it could be addressed in another PR.

Yes, so simply changing `VFOX_HOME` does not seem to be of much use at present. The default 755 permissions still limit this feature.

**Comment by cifaz** [comment]
Can you add command like vfox config store-path=D:/app/vfox

**Comment by cifaz** [comment]
Can you add command like vfox config store-path=D:/app/vfox

**Pull request closed without merging**